src/main/java/uk/gov/hmcts/darts/dailylist/mapper/DailyListRequestMapper.java

```diff
@@ -127,7 +127,20 @@ default List<Defendant> map(HearingStructure.Defendants defendants) {
         return CollectionUtils.emptyIfNull(defendants.getDefendant()).stream().map(this::map).toList();
     }
 
-    Defendant map(DefendantStructure defendant);
+    default Defendant map(DefendantStructure defendant) {
```

Comment:
is the same not possible with:-
```
    @Mappings({
        @Mapping(source = "URN", target = "urn")
    })
```

Reply:
Yes, that's much better, thanks @cmagowan-hmcts